girder/api/v1/folder.py

```diff
@@ -213,21 +213,24 @@ def updateFolderAccess(self, folder, access, publicFlags, public, recurse, progr
         .param('name', 'Name of the folder.', strip=True)
         .param('description', 'Description for the folder.', required=False,
                default='', strip=True)
+        .param('reuseExisting', "Return existing folder if it exists rather than "
```

Comment:
Roughly half of the param descriptions for createFolder are double quotes. Should I changed them all to single?